app/services/task_service.py

```
@@ -2,5 +1,103 @@
+        for task in tasks:
+
+            if task.get('status') not in self.valid_statuses:
+                results['invalid_tasks'].append(task)
+                continue
+            
+
+            processed_task = self._transform_task(task)
+            
+    
+            if 'user_email' in task:
+                if self.email_pattern.match(task['user_email']):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)